crates/runtime/src/lib.rs

```diff
@@ -70,9 +83,16 @@ impl Builder {
             engine,
             module,
             state_builder: SlightCtxBuilder::default(),
+            io_redirects: None,
         })
     }
 
+    /// Set the I/O redirects for the module
+    pub fn set_io(mut self, io_redirects: IORedirects) -> Self {
```

Comment:
Not related to this PR but something we might want to explore futher in the future is that if we think `slight CLI` as a wrapper on `wasmtime CLI`, then obviously we want to expose all the wasmtime configurations to slight CLI as well. This may require us to build something similar to `WasiCtxBuilder` and may call it `SlightCtxBuilder` which can be exposed to the users.

Reply:
Sounds great. I think you mentioned something like this before. We should chat about what it would look like / behave.

Reply:
Created an issue #358 for that. Will resolve this convo